setup.cfg

```diff
@@ -37,6 +37,8 @@ install_requires =
     numpy>=1.17.2,<2
     # omegaconf 2.1+ required for to_object method
     omegaconf>=2.1,<3
+    # packaging 17+ required by pytorch-lightning
+    packaging>=17,<22
```

Comment:
Unfortunately, the only way to reliably parse and compare package versions is using the `packaging` library. We were able to avoid this for pillow because we could use a try-except and try both the new and old syntax. But torchvision uses kwargs so even if you provide a non-existent argument (or simply misspell it) it will be silently ignored. We _could_ only support the brand new version of torch/torchvision and avoid this new dep, but I'd rather avoid that.